monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -0,0 +1,324 @@
+from shapely.geometry import Point, Polygon
+import shapely.geometry
+from pyproj import Geod, Transformer
+import json
+from pathlib import Path
+from typing import List, NamedTuple
+import arrow
+import datetime
+from datetime import datetime, timedelta
+import uuid
+
+class QueryBoundingBox(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+    name: str
+    shape: Polygon
+    time_step_after: int    
```

Comment:
OK, no problems, I have switched it to timestamps, we may need further changes, since the timestep seems a bit more logical from a test director point of view.